docs/apiary.apib

```diff
@@ -458,7 +458,7 @@ Register a new device. In this example, there is already a template (ID 1) which
 
             {"status" : "device has missing fields"}
 
-+ Response 304 (application/json)
++ Response 401 (application/json)
```

Comment:
One important question: when aglio and apiary renders this file with this 304 return code, they say that this must not have any payload. Is this actually the correct code? By the status message, it seems that it would be more appropriate the 401, which is "unauthorized"